stages/org.osbuild.bootupd

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python3
+"""
+Install GRUB on both BIOS and UEFI systems,
+ensuring that your bootloader stays up-to-date.
+
+Bootupd supports updating GRUB and shim for
+UEFI firmware on x86_64 and aarch64,
+and GRUB for BIOS firmware on x86_64.
+The project is deployed in Fedora CoreOS and derivatives
+"""
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util import ostree
+
+SCHEMA_2 = r"""
+"devices": {
+  "type": "object",
+  "additionalProperties": true
+},
+"mounts": {
+  "type": "array"
+},
+"options": {
+  "additionalProperties": true,
```

Comment:
honestly probably because I was copying stuff from another stage? I guess `additionalProperties` just allows for unknown to the schema properties to be specified?

Reply:
Thanks, yeah, it would unknown properties (which is sometimes what we want and sometimes not :)